### PR TITLE
Removed statements not used by the tests

### PR DESCRIPTION
**Pull request by elopio**
LP: #1617149

### CODE REVIEW

File: snapcraft/tests/test_options.py [hunk 1]
```diff
@@ -21,44 +21,44 @@
...
-            snapcraft.ProjectOptions()
-        except KeyError:
-            self.fail('Expected arm to be supported')
 
-    @mock.patch('platform.machine', return_value='aarch64')
-    def test_arm64_support(self, mock_platform_machine):
-        try:
-            snapcraft.ProjectOptions()
-        except KeyError:
-            self.fail('Expected arm64 to be supported')
```
Comment (elopio) [inline snapcraft/tests/test_options.py]: I'm sorry @kyrofa, I couldn't deal with these tests that didn't have 100% coverage.

**Comment by elopio** [comment]
retest this please

File: snapcraft/tests/test_options.py [hunk 2]
```diff
@@ -21,44 +21,44 @@
...
-            self.fail('Expected ppc to be supported')
-
-    @mock.patch('platform.machine', return_value='s390x')
-    def test_s390x_support(self, mock_platform_machine):
-        try:
-            snapcraft.ProjectOptions()
-        except KeyError:
-            self.fail('Expected s390x to be supported')
+    @mock.patch('platform.machine')
+    def test_amd64_support(self, mock_platform_machine):
```
Comment (sergiusens) [inline snapcraft/tests/test_options.py]: the test name is now innacurate ;-)

**Comment by sergiusens** [comment]
I +1 this, it fits my style. Just need a test rename in there. And yay coverage.

I hope @kyrofa is ok with this.

**Pull request merged**